Compiler Error

Are you guys getting these compiler errors? @justbuchanan @jgkamat 
../logging/log_converter.cpp:194:13: error: variable 'yellowScore' is used uninitialized whenever 'if' condition is false [-Werror,-Wsometimes-uninitialized]
        if (currentFrame->raw_refbox_size() > 0) {
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../logging/log_converter.cpp:213:21: note: uninitialized use occurs here
                 << yellowScore << BAR << blueScore << BAR << yellowGoalie
                    ^~~~~~~~~~~
../logging/log_converter.cpp:194:9: note: remove the 'if' if its condition is always true
        if (currentFrame->raw_refbox_size() > 0) {
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../logging/log_converter.cpp:189:24: note: initialize the variable 'yellowScore' to silence this warning
        int yellowScore;
                       ^
                        = 0
../logging/log_converter.cpp:194:13: error: variable 'blueScore' is used uninitialized whenever 'if' condition is false [-Werror,-Wsometimes-uninitialized]
        if (currentFrame->raw_refbox_size() > 0) {
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../logging/log_converter.cpp:213:43: note: uninitialized use occurs here
                 << yellowScore << BAR << blueScore << BAR << yellowGoalie
                                          ^~~~~~~~~
../logging/log_converter.cpp:194:9: note: remove the 'if' if its condition is always true
        if (currentFrame->raw_refbox_size() > 0) {
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../logging/log_converter.cpp:188:22: note: initialize the variable 'blueScore' to silence this warning
        int blueScore;
                     ^
                      = 0
../logging/log_converter.cpp:194:13: error: variable 'yellowGoalie' is used uninitialized whenever 'if' condition is false [-Werror,-Wsometimes-uninitialized]
        if (currentFrame->raw_refbox_size() > 0) {
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../logging/log_converter.cpp:213:63: note: uninitialized use occurs here
                 << yellowScore << BAR << blueScore << BAR << yellowGoalie
                                                              ^~~~~~~~~~~~
../logging/log_converter.cpp:194:9: note: remove the 'if' if its condition is always true
        if (currentFrame->raw_refbox_size() > 0) {
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../logging/log_converter.cpp:192:25: note: initialize the variable 'yellowGoalie' to silence this warning
        int yellowGoalie;
                        ^
                         = 0
../logging/log_converter.cpp:194:13: error: variable 'blueGoalie' is used uninitialized whenever 'if' condition is false [-Werror,-Wsometimes-uninitialized]
        if (currentFrame->raw_refbox_size() > 0) {
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../logging/log_converter.cpp:214:28: note: uninitialized use occurs here
                 << BAR << blueGoalie << endl;
                           ^~~~~~~~~~
../logging/log_converter.cpp:194:9: note: remove the 'if' if its condition is always true
        if (currentFrame->raw_refbox_size() > 0) {
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../logging/log_converter.cpp:191:23: note: initialize the variable 'blueGoalie' to silence this warning
        int blueGoalie;
                      ^
                       = 0

--- a/logging/log_converter.cpp
+++ b/logging/log_converter.cpp
@@ -185,11 +185,11 @@ int main(int argc, char* argv[]) {
         string stage;
         string command;
 
-        int blueScore;
-        int yellowScore;
+        int blueScore = 0;
+        int yellowScore = 0;
 
-        int blueGoalie;
-        int yellowGoalie;
+        int blueGoalie = 0;
+        int yellowGoalie = 0;
 
         if (currentFrame->raw_refbox_size() > 0) {
             SSL_Referee referee = currentFrame->raw_refbox(0);

